### PR TITLE
✨ adds elasticsearch resolver for dashboard

### PR DESCRIPTION
**Pull request by hlminh2000**

### CODE REVIEW

File: src/schemas/ProgramDonorSummary/resolvers/summaryStats.ts [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+        esb
+          .termsQuery()
+          .field(EsDonorDocumentField.processingStatus)
+          .values(['PROCESSING'] as ElasticsearchDonorDocument['processingStatus'][]),
+      ),
+      filterAggregation('donorsWithReleasedFilesCount').filter(
+        esb
+          .termsQuery()
+          .field(EsDonorDocumentField.processingStatus)
+          .values(['COMPLETE'] as ElasticsearchDonorDocument['processingStatus'][]),
```
Comment (jaserud) [inline src/schemas/ProgramDonorSummary/resolvers/summaryStats.ts]: Shouldn't it be an aggregate of donors with `relaseStatus` === `COMPLETE` or `PARTIAL`?? Or am I misunderstanding this field.

File: .schema.env [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+EGO_PUBLIC_KEY=
+
+# Program Service
+PROGRAM_SERVICE_ROOT=
+
+# Clinical
+CLINICAL_SERVICE_ROOT=
+SUBMISSION_TEMPLATE_PATH=
+
+# Lafla
```
[the remark this answers is not in the document]
Comment (hlminh2000) [reply .schema.env]: =))) Kafka is just glorified Lafla on steroid didn't you know? lol

File: src/schemas/ProgramDonorSummary/resolvers/summaryStats.ts [hunk 3]
```diff
@@ -39,31 +40,34 @@ const programDonorSummaryStatsResolver: (
...
         esb
           .termsQuery()
           .field(EsDonorDocumentField.processingStatus)
-          .values(['PROCESSING'] as ElasticsearchDonorDocument['processingStatus'][]),
+          .values([DonorMolecularDataProcessingStatus.PROCESSING]),
       ),
       filterAggregation('donorsWithReleasedFilesCount').filter(
         esb
           .termsQuery()
           .field(EsDonorDocumentField.processingStatus)
```
Comment (jaserud) [inline src/schemas/ProgramDonorSummary/resolvers/summaryStats.ts]: `releaseStatus`...

Comment (hlminh2000) [reply src/schemas/ProgramDonorSummary/resolvers/summaryStats.ts]: oops!

**Pull request merged**